packages/grafana-prometheus/src/language_provider.test.ts

```diff
@@ -301,6 +305,50 @@ describe('Language completion provider', () => {
           end: toPrometheusTimeString,
           'match[]': 'interpolated-metric',
           start: fromPrometheusTimeString,
+          limit: DEFAULT_SERIES_LIMIT,
+        },
+        undefined
+      );
+    });
+
+    it('should have a default limit parameter', () => {
+      const languageProvider = new LanguageProvider({
+        ...defaultDatasource,
+        // interpolateString: (string: string) => string.replace(/\$/, 'interpolated-'),
```

Comment:
Is this needed?

Reply:
I updated the test with something more meaningful. I made the test use the 'none' parameter to remove the limit.